examples/aws-s3-folder/Main.scala

```diff
@@ -61,7 +61,7 @@ val siteDir = "www"
   )
 
   // For each file in the directory, create an S3 object stored in `siteBucket`
-  val uploads: Array[Output[s3.BucketObject]] = File(siteDir).listFiles().map { file =>
+  val uploads: Output[List[s3.BucketObject]] = File(siteDir).listFiles().map { file =>
```

Comment:
nit: if it's an example then maybe we should use traverse, since `.map(...).sequence == .traverse(...)`?